### ISSUE TITLE
Should "accessibility features of software" have a link to the WCAG2ICT key term "software"?

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#476]
In a few definitions...Since links added to the key term "software" are being removed from WCAG language because WCAG doesn't have a "software" definition (see Issue #412), should cases where the definitions have word substitutions such as "accessibility features of software" should there be a link to "software" in that phrase? For example, in the current editor's draft:

>**programmatically determined (programmatically determinable)**
>determined by [software](https://deploy-preview-471--wcag2ict.netlify.app/#software) from author-supplied data provided in a way that different **[[assistive technologies](#dfn-assistive-technologies) and accessibility features of software]** can extract and present this information to users in different modalities

The above text has been changed by PR 471 to remove the "software" link because the original WCAG text doesn't have a link because no definition of "software" exists in WCAG. 
>**programmatically determined (programmatically determinable)**
>determined by software from author-supplied data provided in a way that different **[[assistive technologies](#dfn-assistive-technologies) and accessibility features of software]** can extract and present this information to users in different modalities

Should it instead be moved to our replacement phrase?

>**programmatically determined (programmatically determinable)**
>determined by software from author-supplied data provided in a way that different **[[assistive technologies](#dfn-assistive-technologies) and accessibility features of [software](https://deploy-preview-471--wcag2ict.netlify.app/#software)]** can extract and present this information to users in different modalities

**Comment by pday1** [comment]
Pull request added with proposed change: https://github.com/w3c/wcag2ict/pull/482

**Comment by maryjom** [comment]
This is purely editorial - doesn't require TF review. Changes have been merged.
